param/tasks.yaml

```diff
@@ -59,9 +63,16 @@ drone_landing_gear_width: .775
 
 # HEIGHT HOLDER
 # p-term for height holder helper for Z height
-p_term_height_hold_z: .90
+p_term_height_hold_z: 2.0
 # maximum z error for heigth holder
 max_z_error: 1.0
+# distance from setpoint that will allow deadzone mode to begin
+deadzone_height_hold_z: 0.05
```

Comment:
Units. Also, why did you choose these names? what does `deadzone` mean?

Reply:
Deadzone is an area where the control output is 0. I just came up with the names.